### PR TITLE
SFA Remap

### PR DESCRIPTION
**Pull request by SophieMint**
- Remaps the SFA Ship into something more lore fitting. https://imgur.com/a/p8H9Flj
- This gives the shuttle a lathe. The ship is really busted up and ship combat will ruin it further, so it provides a bit more support. It also allows them to print what they need, albeit with a very limited supply of metal. In return this removes the combat shotgun so slugs can't be spammed, and gives them an additional AR instead - bringing them more up to par with more recent ones. Updates the description courtesy of Danse.
- I have almost certainly made a bunch of mistakes with this PR. It was a bit messy to get working.
- My previous one had a ton of issues, had to close and reopen as I couldn't figure out what was going wrong with my repo. Sorry!
- Also adds functional atmos and power using the fancy new fusion reactors.

### CODE REVIEW

**Comment by SophieMint** [comment]
!wip

**Comment by SophieMint** [comment]
!review

File: html/changelogs/PeppermintSFA.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "Remaps the SFA away ship"
```
Comment (SleepyGemmy) [inline html/changelogs/PeppermintSFA.yml:41]:
```suggestion
  - maptweak: "Remaps the SFA away ship."
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
image
don't map curtains inside of windows.

image
don't map dirt decals on space tiles.

image
this turf should be passthrough.

image
this tile is going to vent the ship. since the room to the north of it is.

image
random surgery holosigns.

image
the "white" tiles in this room should be `/turf/space/dynamic`.

image
theres a floor decal under this window.

image
floor decals misaligned.

image
turfs underneath the girders should be `/turf/space/dynamic`.

image
windows on corners like this is bad praxis.

image
missing turf.

image
?

**Comment by SophieMint** [comment]
> don't map curtains inside of windows. image

I think I got most of 'em

**Review by Alberyk — APPROVED** [review]

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
generally, if a turf has nothing on it and its in space, use the passthrough turf.
if a turf has something on it and its in space, use `/turf/space/dynamic`.

something under this window.
image

holosign still here.
image

turf still missing.
image

**Review by NonQueueingMatt — APPROVED** [review]

**Review by SleepyGemmy — COMMENTED** [review]
no tile underneath.
image

File: maps/away/ships/sol_pirate/sfa_patrol_ship.dm [hunk 2]
```diff
@@ -10,12 +10,50 @@
...
+	name = "SFA Brig"
+
+/area/ship/sfa_patrol_ship/Engine1
+	name = "SFA Engine One"
+
+/area/ship/sfa_patrol_ship/Engine2
+	name = "SFA Engine Two"
+
+/area/ship/sfa_patrol_ship/Suit_Storage
+	name = "SFA Suit Storage"
```
Comment (SleepyGemmy) [inline maps/away/ships/sol_pirate/sfa_patrol_ship.dm:55]: paths should be lowercase.

Comment (SleepyGemmy) [inline maps/away/ships/sol_pirate/sfa_patrol_ship.dm:55]:
```suggestion
/area/ship/sfa_patrol_ship/sfa_armory
	name = "SFA Armory"

/area/ship/sfa_patrol_ship/engineering
	name = "SFA Engineering"

/area/ship/sfa_patrol_ship/telecoms
	name = "SFA Telecoms"

/area/ship/sfa_patrol_ship/treasureroom
	name = "SFA Treasure Room"

/area/ship/sfa_patrol_ship/bridge
	name = "SFA Corvette Bridge"

/area/ship/sfa_patrol_ship/ammo_closet
	name = "SFA Ammo Closet"

/area/ship/sfa_patrol_ship/quarters
	name = "SFA Crew Quarters"

/area/ship/sfa_patrol_ship/officer
	name = "SFA Officer Quarters"

/area/ship/sfa_patrol_ship/brig
	name = "SFA Brig"

/area/ship/sfa_patrol_ship/engine1
	name = "SFA Engine One"

/area/ship/sfa_patrol_ship/engine2
	name = "SFA Engine Two"

/area/ship/sfa_patrol_ship/suit_storage
	name = "SFA Suit Storage"
```

**Review by SleepyGemmy — APPROVED** [review]

File: maps/away/ships/sol_pirate/sfa_patrol_ship.dm [hunk 3]
```diff
@@ -10,12 +10,51 @@
...
+	name = "SFA Brig"
+
+/area/ship/sfa_patrol_ship/engine1
+	name = "SFA Engine One"
+
+/area/ship/sfa_patrol_ship/engine2
+	name = "SFA Engine Two"
+
+/area/ship/sfa_patrol_ship/suit_storage
+	name = "SFA Suit Storage"
```
Comment (SleepyGemmy) [inline maps/away/ships/sol_pirate/sfa_patrol_ship.dm:56]:
```suggestion
/area/ship/sfa_patrol_ship/SFA_Armory
	name = "SFA Armory"

/area/ship/sfa_patrol_ship/Engineering
	name = "SFA Engineering"

/area/ship/sfa_patrol_ship/Telecoms
	name = "SFA Telecoms"

/area/ship/sfa_patrol_ship/TreasureRoom
	name = "SFA Treasure Room"

/area/ship/sfa_patrol_ship/Bridge
	name = "SFA Corvette Bridge"

/area/ship/sfa_patrol_ship/Ammo_Closet
	name = "SFA Ammo Closet"

/area/ship/sfa_patrol_ship/Quarters
	name = "SFA Crew Quarters"

/area/ship/sfa_patrol_ship/Officer
	name = "SFA Officer Quarters"

/area/ship/sfa_patrol_ship/Brig
	name = "SFA Brig"

/area/ship/sfa_patrol_ship/Engine1
	name = "SFA Engine One"

/area/ship/sfa_patrol_ship/Engine2
	name = "SFA Engine Two"

/area/ship/sfa_patrol_ship/Suit_Storage
	name = "SFA Suit Storage"
```

**Pull request merged**